rollup.config.js

```diff
@@ -68,7 +68,7 @@ const cli_config = {
 };
 
 if (process.env.PEGGY_CLI_DEBUG) {
-  cli_config.output.sourcemap = true;
+  cli_config.output.sourcemap = "inline";
```

Comment:
jest can't handle external sourceMaps.  Something to consider for our sourcemap output.  We could, for example, make a filename of "inline" be magical, and the default.

Reply:
Maybe. Honestly, never used source maps, don't known how convenient it will be

Reply:
It's particularly convenient if people are using broken tools like jest.  It will make more sense once you run `npm run coverage:bin`, and look at the end of the `bin/peggy.js` file.